sale_order_general_discount_triple/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2022 Ooops - Ashish Hirpara
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Sale Order General Discount Triple",
+    "summary": "General discount per sale order with triple",
+    "version": "14.0.1.0.0",
+    "category": "Sales",
+    "website": "https://github.com/OCA/sale-workflow",
+    "author": "Ashish Hirpara, Ooops, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "contributors": ["Ashish-Hirpara"],
```

Comment:
@rousseldenis even for a new module? so who would be the author? :)

Reply:
> @AshishHirapara Thanks for this.
> 
> Minor comment.
> 
> As this could be critical flows,could you add tests ?

Yes, I am working on it. Soon commit it.

Reply:
> @rousseldenis even for a new module? so who would be the author? :)

@francesco-ooops

The contributors does not exist in manifest: https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py

Reply:
done